data/json/monstergroups/zombie_upgrades.json

```diff
@@ -85,6 +85,16 @@
       { "monster": "mon_skeleton", "weight": 50 }
     ]
   },
+  {
+    "type": "monstergroup",
+    "name": "GROUP_FIREMAN_UPGRADE",
+    "//": "Masters pick from here when upgrading.  Note that 'freq' units are tenth of a percent, with default filling in the gap.",
```

Comment:
Nitpick, but this comment is outdated. The `freq` units are now called `weight` since #52288, and I'm not sure about "tenth of a percent" part.

Reply:
mb I just cargo culted it. Should I remove that line entirely?

Reply:
Yeah, I think it's unneeded.